### PR TITLE
Add traces from PicoScope

### PR DESCRIPTION
**Pull request by weiglszonja**
This PR adds the raw voltage traces from a PicoScope 6 acquisition system.

Description from #1 :
Simultaneous traces (velocity from rotary encoder, trigger signals for reward, air puff and light stimuli delivery, licking from a lick sensor, fluorescence detected by photomultiplier tubes (PMTs) from one or two optic fibers and output from waveform generator used to alternate 405-nm and 470-nm illumination every 10 ms) were collected at 4 kHz by a PicoScope 6 data acquisition system.

### CODE REVIEW

File: src/dombeck_lab_to_nwb/azcorra2023/azcorra2023_metadata.yaml [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
+  institution: Northwestern University
   lab: Dombeck
   experimenter:
-    - Last, First Middle
-    - Last, First Middle
-Subject:
-  species: Rattus norvegicus
-  age: TBD  # in ISO 8601, such as "P1W2D"
-  sex: TBD  # One of M, F, U, or O
+    - He, Qianzi
```
Comment (weiglszonja) [inline src/dombeck_lab_to_nwb/azcorra2023/azcorra2023_metadata.yaml:9]: TODO: modify experimenter name

File: src/dombeck_lab_to_nwb/azcorra2023/interfaces/picoscope_recordinginterface.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        custom_channel_names = [
+            "chMov",
+            "chRed",
+            "chGreen",
+            "Light",
+            "ch470",
+            "Reward",
+            "Licking",
+            "AirPuff",
+        ]
```
Comment (weiglszonja) [inline src/dombeck_lab_to_nwb/azcorra2023/interfaces/picoscope_recordinginterface.py:52]: Before I'm making the changes, do you think @alessandratrapani I should follow the same design as you had and create an `ElectricalSeries` for each channel here?

Keep in mind for this format I have to concatenate recordings, which I'm doing this way only once, but I'm adding only one `ElectricalSeries` object.

Comment (weiglszonja) [reply src/dombeck_lab_to_nwb/azcorra2023/interfaces/picoscope_recordinginterface.py:60]: Bearing this is mind, I uploaded a prototype file [20200129-0001.nwb](https://s3.console.aws.amazon.com/s3/object/catalystneuro-processing?region=us-east-2&bucketType=general&prefix=dombeck-lab-to-nwb/20200129-0001.nwb)

Comment (alessandratrapani) [reply src/dombeck_lab_to_nwb/azcorra2023/interfaces/picoscope_recordinginterface.py:60]: Actually, now that I see your PR, I think that we should organize this kind of signals differently. Synch signals should all go together in the same ElectricalSeries object, but here the only signal that can be considered a sync signal is  "ch470". 

The following are actual analog recordings:
- A = velocity (chMov)
- B = fiber 2 fluorescence (from here on called channel red - chR)
- C = fiber 1 fluorescence (channel green - chG)
- G = licking sensor output

And the rest are stimulus delivery signals:
- D = light stimulus trigger
- F = reward delivery trigger
- H = air puff delivery trigger

The easiest approach is to create an ElectricalSeries object for each channel, but then you lose the reference to the electrodes table, don't you? (I notice this in my stub [file](https://catalystneuro-processing.s3.us-east-2.amazonaws.com/higley-lab-to-nwb/11222019_grabAM05_spont_onlyTTLSignals.nwb))

There must be examples from past conversion projects that we can use as references.

Comment (weiglszonja) [reply src/dombeck_lab_to_nwb/azcorra2023/interfaces/picoscope_recordinginterface.py:60]: I agree, it definitely feels weird to have `Electrodes` and `ElectroGroup` for these signals. 

> D-F-H should go in intervals I guess

I found how G, D, F, H (licking, light, reward, air puff) binary signals are processed [here](https://github.com/DombeckLab/Azcorra2023/blob/2819bd5b7a6021243c44dfd45b5b25fd24ae8122/Fiber%20photometry%20data%20analysis/Data%20pre%20processing/selectSignals_paper.m#L109C12-L109C48) from lab. 
I can use this threshold to identify the onset and offset of events probably.

Comment (weiglszonja) [reply src/dombeck_lab_to_nwb/azcorra2023/interfaces/picoscope_recordinginterface.py:60]: Update from the team meeting; we are going to store the analog signals as `ElectricalSeries` for now and the binary signals as events using `ndx-events`.

**Comment by weiglszonja** [comment]
@alessandratrapani

I'm adding an `ElectricalSeries` for the velocity and fluorescence signals as the follows:

Screenshot 2024-03-13 at 15 49 50 (2)

The binary signals (licking (if not empty), reward, air puff, light) are added to an `ndx_events.AnnotatedEventsTable` `DynamicTable` with description:

Screenshot 2024-03-13 at 15 55 24 (2)

Uploaded [20200129-0001.nwb](https://s3.console.aws.amazon.com/s3/object/catalystneuro-processing?region=us-east-2&bucketType=general&prefix=dombeck-lab-to-nwb/20200129-0001.nwb) that contains these changes.

**Comment by alessandratrapani** [comment]
I was looking at the ndx-event documentation, there is a subtype of LabeledEvents, called TTLs. Why don't we used that one, for Ch405 and Ch470? I think it's ok to keep the Light stimulus trigger, the Airpuff trigger and the Reward delivery trigger as AnnotatedEventsTable

**Comment by alessandratrapani** [comment]
Ah well, let's keep things easy then. I was suggesting the TTL type because for Ch405 and Ch470 the onset duration might be relevant, and TTLs type seemed just right.

**Review by alessandratrapani — COMMENTED** [review]
LGTM!

**Review by alessandratrapani — APPROVED** [review]

**Pull request merged**